### PR TITLE
Remove outdated HA sync, reuse HA Sync Plugin info

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by MichaelBaj — APPROVED** [review]

**Review by tcarroll25 — DISMISSED** [review]

File: docs/plugin_ha_sync_redundancy.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (agrawalkaushik) [inline docs/plugin_ha_sync_redundancy.mdx:1]: not sure what the `.mdx` format is for but this plugin is no longer necessary so please feel free to deprecate and remove all the documentation related to this plugin.

Comment (MichaelBaj) [reply docs/plugin_ha_sync_redundancy.mdx:1]: `.mdx` file is a markdown file that support javascript. Note the use of the `import` directives in the file.
Kaushik, I think I am missing something significant here. Is the plugin no longer a plugin and now there is native support for HA config without the use of the plugin? Why would the plugin no longer be necessary?

Comment (agrawalkaushik) [reply docs/plugin_ha_sync_redundancy.mdx:1]: If my memory serves me right the plugin was used before we had team interfaces. I talked a bit more with Lane and it looks like there are some other alternative use cases at bunch of customers. So please disregard my comment above the plugin not being necessary.

File: docs/plugin_ha_sync_redundancy.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+## Configuration Snippet
+
+### Redundant HA Sync Interface
+<RedundantHASync/> 
+
+## Use Cases
+<HASyncUseCases/>
+
+## Troubleshooting
+<HASyncTroubleshooting/>
```
Comment (agrawalkaushik) [inline docs/plugin_ha_sync_redundancy.mdx:23]:
```suggestion
<HASyncTroubleshoot/>
```

Without the change, the troubleshooting section is not getting rendered.

Comment (agrawalkaushik) [reply docs/plugin_ha_sync_redundancy.mdx:23]: side note - just like we catch broken links, I wonder if docusaurus will let us catch this!

**Review by agrawalkaushik — CHANGES_REQUESTED** [review]
lgtm!

Just need to fix the troubleshooting link!

File: docs/plugin_ha_sync_redundancy.mdx (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/plugin_ha_sync_redundancy.mdx:23]: Nice catch Kaushik. Docusaurus did not catch that. Nor did I in my local build.

**Review by MichaelBaj — APPROVED** [review]

File: docs/plugin_ha_sync_redundancy.mdx (hunk 2 above)
Comment (agrawalkaushik) [reply docs/plugin_ha_sync_redundancy.mdx]: I usually spin up a local build to review doc changes... My click on the Troubleshooting page didn't work 😄

**Pull request merged**